### PR TITLE
tickets/DM 40775

### PR DESCRIPTION
**Pull request by couger01**
- Cycle 32 Revision 4
- In rapid analysis build, install rubin-libradtran.
- In rapid analysis build, add environment variables to limit the number of threads/processes used by pipeline tasks to 1 and to set the MPL backend to Agg.
- DM-40660: Add audio_broadcaster to cycle build
- DM-39426: Add oods to cycle build
- DM-39427: Add HeaderService to cycle build
- Update version history

### CODE REVIEW

**Review by wvreeven — APPROVED** [review]
Looks good to me.

**Comment by mareuter** [comment]
We don't need the `atheaderservice` and `ccheaderservice` specific builds. The base one server all our needs.

**Review by wvreeven — CHANGES_REQUESTED** [review]
Please re-request my review when you're done with the latest changes.

File: build/audio_broadcaster/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+FROM ${hub}/deploy-env:${cycle}
+
+LABEL maintainer Eric Coughlin <ecoughlin@lsst.org>
+
+WORKDIR /home/saluser/
+
+ARG ts_audio_broadcaster
+ENV MICROPHONE_SERVER_HOST auxtel-rpi-audio01.cp.lsst.org
+ENV MICROPHONE_SERVER_PORT 4444
```
Comment (wvreeven) [inline build/audio_broadcaster/Dockerfile:12]: These two env vars need to be defined at deploy time so remove them here.

**Review by wvreeven — DISMISSED** [review]
Please remove the two env vars as indicated.

File: build/headerservice/setup_HeaderService.env
Comment (tribeiro) [inline build/headerservice/setup_HeaderService.env:1]: Is this file being used at all? I think you can remove it no?

File: build/headerservice/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    git clone https://github.com/lsst-dm/HeaderService -b ${headerservice} && \
+    cd HeaderService && \
+    export PYTHONPATH=$PYTHONPATH:$CONDA_PREFIX/HeaderService/python &&\
+    python setup.py install --prefix=$CONDA_PREFIX/HeaderService --install-lib=$CONDA_PREFIX/HeaderService/python --install-data $CONDA_PREFIX/HeaderService &&\
+    python setup.py install_data --install-dir=$CONDA_PREFIX/HeaderService
+
+# Extra utils init files
+COPY --chown=saluser:saluser setup_HeaderService.env /home/saluser/
+
+COPY start_ATHeaderService.sh /home/saluser/
```
Comment (tribeiro) [inline build/headerservice/Dockerfile:25]: Is this file needed at all?

File: build/headerservice/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+COPY start_CCHeaderService.sh /home/saluser/
```
Comment (tribeiro) [inline build/headerservice/Dockerfile:27]: Is this file needed?

File: build/headerservice/start_ATHeaderService.sh
Comment (tribeiro) [inline build/headerservice/start_ATHeaderService.sh:1]: I think you can remove this file

File: build/headerservice/start_CCHeaderService.sh
Comment (tribeiro) [inline build/headerservice/start_CCHeaderService.sh:1]: I think you can remove this file

**Review by tribeiro — COMMENTED** [review]
I have some additional comments. I think there are some extra files in the header service that can be removed. It is a bit confusing to have those files added as it gives the impression they are being used instead of the correct ones you actually use.

**Review by tribeiro — APPROVED** [review]
Looks good!

**Pull request merged**